vega_sim/scenario/fuzzed_markets/agents.py

```diff
@@ -173,6 +218,38 @@ def _select_order_id(self):
         else:
             return None
 
+    def finalise(self):
+        if not self.__class__.OUTPUTTED:
```

Comment:
Can we add an `output_plot_on_finalise` flag to the class then bring the plotly import actually inside this function? Would avoid having to have a top level import for plotting stuff when people may not always want/need